src/performance.rs

```diff
@@ -46,6 +55,7 @@ impl ModuleDef for PerformanceModule {
     fn declare(declare: &mut Declarations) -> Result<()> {
         declare.declare("timeOrigin")?;
         declare.declare("now")?;
+        declare.declare("toJSON")?;
```

Comment:
Use `PredefinedAtom::toJSON`

Reply:
hi @richarddavison
Thank you for quick review, and let me check.

Correcting line 58 still results in a compile error.

```log
error[E0277]: the trait bound `Vec<u8>: std::convert::From<rquickjs::atom::PredefinedAtom>` is not satisfied
   --> src/performance.rs:59:25
    |
59  |         declare.declare(PredefinedAtom::ToJSON)?;
    |                 ------- ^^^^^^^^^^^^^^^^^^^^^^ the trait `std::convert::From<rquickjs::atom::PredefinedAtom>` is not implemented for `Vec<u8>`, which is required by `rquickjs::atom::PredefinedAtom: Into<Vec<u8>>`
    |                 |
    |                 required by a bound introduced by this call
    |
```

It seems to work if line 45 is modified as follows.
```rust
performance.set(PredefinedAtom::ToJSON, Func::from(to_json))?;
```

Sorry, I don't understand how PredefinedAtom::ToJSON works, so I can't judge if this fix is correct.

Reply:
Sorry I meant line 45

Reply:
We have corrected the problem, please review.